mindee/client.py

```diff
@@ -178,6 +179,23 @@ def enqueue(
             cropper,
         )
 
+    def load_prediction(
+        self, product_class: Type[Inference], local_response: LocalResponse
+    ) -> Union[AsyncPredictResponse, PredictResponse]:
+        """
+        Load a prediction.
+
+        :param product_class: Class of the product to use.
+        :param local_response: Local response to load.
+        :return:
+        """
+        try:
+            if local_response.as_dict.get("job"):
+                return AsyncPredictResponse(product_class, local_response.as_dict)
+            return PredictResponse(product_class, local_response.as_dict)
+        except KeyError as exc:
```

Comment:
there should never be a key error as you're using `get`.

Reply:
This isn't for the `get`. The keyerror occurs when we try to load something without a top-level document object.